Core/Src/main.c

```diff
@@ -117,13 +131,21 @@ int main(void)
 
   /* Infinite loop */
   /* USER CODE BEGIN WHILE */
+  HAL_TIM_PWM_Start(&htim2, TIM_CHANNEL_3);
+  HAL_GPIO_WritePin(SERVO_EN_GPIO_Port, SERVO_EN_Pin, GPIO_PIN_SET);
   float n = 1.234f;
   while (1)
   {
-	  HAL_GPIO_TogglePin(STATUS_IND_GPIO_Port, STATUS_IND_Pin);
-	  HAL_GPIO_TogglePin(WARN_IND_GPIO_Port, WARN_IND_Pin);
+	  // Checks servo continuity then performs test actuation
 	  send_can_msg(&n, sizeof(n));

```

Comment:
if this fails it do something to indicate that it also failed, because can could break while continuity is working, which would indicate no errors

Reply:
Yea, that's something that we can implement later down the road.  This PR is just to set up the basics of controlling the servo so that Charlie can build a state machine on top of it.